rotkehlchen/db/dbhandler.py

```diff
@@ -3494,3 +3494,39 @@ def get_associated_locations(self) -> Set[Location]:
         if cursor.fetchone()[0] >= 1:
             locations.add(Location.BALANCER)
         return locations
+
+    def get_timed_balances(self, timestamp: Timestamp) -> List[SingleDBAssetBalance]:
```

Comment:
There is already code in dbhandler that queries timed balances here: https://github.com/rotki/rotki/blob/963c13219c308f1efd19724e6793a1013482ddc5/rotkehlchen/db/dbhandler.py#L2770

Why not use that?

Reply:
The issue with using `query_timed_balances` is the fact that it defines an `asset` parameter. The constraint with that is having to add multiple lists together.

I might be wrong though.